socorro/middleware/middleware_app.py

```diff
@@ -551,7 +552,7 @@ def GET(self, *args, **kwargs):
         except AttributeError:
             try:
                 if (method_name == 'post' and
-                    getattr(instance, 'create', None)):
+                   getattr(instance, 'create', None)):
```

Comment:
I'm not sure why I did that change. I'm normally not a stickler for pep8 in the non-webapp files. But this was the last change necessary for it to be compliant.